.docker/base/Dockerfile

```diff
@@ -4,33 +4,47 @@ LABEL maintainer "Irina Spiridonova <irinasp@microsoft.com>; Darren Gehring <dar
 USER everest
 WORKDIR ${MYHOME}
 
-#PAT token should be provided in the docker run command or as a variable in VSTS build definition 
-ENV PAT $PAT
+ENV CI_LOGS ${MYHOME}"/ci-logs/"
```

Comment:
As a stylistic note, when you know that the string does not contain any spaces, it's fine to omit the double-quotes.